When compiling the project here is the errors I get : 

```
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 9.285 s
[INFO] Finished at: 2018-07-31T15:15:40+02:00
[INFO] Final Memory: 41M/312M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project rss: Compilation failure: Compilation failure: 
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[87,16] '(' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[87,28] ';' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[87,35] <identifier> expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[87,42] ';' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[87,43] ')' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[88,2] illegal start of expression
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[88,8] ';' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[88,4] variable declaration not allowed here
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[88,10] not a statement
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[88,13] ';' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[89,10] not a statement
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[89,11] ';' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[90,3] illegal start of expression
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[91,1] illegal start of expression
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[109,4] '(' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[109,16] ';' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[109,23] <identifier> expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[109,30] ';' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[109,31] ')' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[110,2] illegal start of expression
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[110,8] ';' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[110,4] variable declaration not allowed here
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[110,10] not a statement
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[110,13] ';' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[111,10] not a statement
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[111,11] ';' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[112,3] illegal start of expression
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[113,1] illegal start of expression
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[131,25] '(' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[131,37] ';' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[131,44] <identifier> expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[131,51] ';' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[131,52] ')' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[132,2] illegal start of expression
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[132,8] ';' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[132,4] variable declaration not allowed here
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[132,10] not a statement
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[132,13] ';' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[133,10] not a statement
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[133,11] ';' expected
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[134,3] illegal start of expression
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[135,1] illegal start of expression
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[239,2] reached end of file while parsing
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[129,9] 'try' without 'catch', 'finally' or resource declarations
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[240,1] reached end of file while parsing
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[107,9] 'try' without 'catch', 'finally' or resource declarations
[ERROR] /Users/loom/java/projects/dx-modules/rss/src/main/java/org/jahia/modules/rss/RSSUtil.java:[85,9] 'try' without 'catch', 'finally' or resource declarations
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

```

--- a/src/main/java/org/jahia/modules/rss/RSSUtil.java
+++ b/src/main/java/org/jahia/modules/rss/RSSUtil.java
@@ -106,7 +106,8 @@ public class RSSUtil {
         SyndFeed feed = null;
         try {
             final SyndFeedInput input = new SyndFeedInput();
-for loading RSS feeds. * User: ktlili
+/** for loading RSS feeds. 
+ * User: ktlili
  * Date: Jan 5, 2010
  * Time: 4:07:04 PM
  */
@@ -128,7 +129,8 @@ public class RSSUtil {
         SyndFeed feed = null;
         try {
             final SyndFeedInput input = new SyndFeedInput();
-                     for loading RSS feeds. * User: ktlili
+ /** for loading RSS feeds. 
+ * User: ktlili
  * Date: Jan 5, 2010
  * Time: 4:07:04 PM
  */

--- a/src/main/java/org/jahia/modules/rss/RSSUtil.java
+++ b/src/main/java/org/jahia/modules/rss/RSSUtil.java
@@ -64,77 +64,6 @@ import java.net.URL;
  * Time: 4:07:04 PM
  */
 public class RSSUtil {
-
-    private static final transient Logger logger = org.slf4j.LoggerFactory.getLogger(RSSUtil.class);
-    
-    private SyndFeed feed;
-    
-    private boolean loaded;
-
-    private String url;
-
-    private static int timeoutInMs = 4000; // default value -> to be loaded from Jahia properties? (global timeout config)
-    
-    /**
-     * Get a SyndFeed from an url
-     * @param url
-     * @return
-     */
-    public static SyndFeed loadSyndFeed(String url){
-        SyndFeed feed = null;
-        try {
-            final SyndFeedInput input = new SyndFeedInput();
-            for loading RSS feeds. * User: ktlili
- * Date: Jan 5, 2010
- * Time: 4:07:04 PM
- */
-public class RSSUtil {
-    private static final transient Logger logger = org.slf4j.LoggerFactory.getLogger(RSSUtil.class);
-    
-    private SyndFeed feed;
-    
-    private boolean loaded;
-    private String url;
-    private static int timeoutInMs = 4000; // default value -> to be loaded from Jahia properties? (global timeout config)
-    
-    /**
-     * Get a SyndFeed from an url
-     * @param url
-     * @return
-     */
-    public static SyndFeed loadSyndFeed(String url){
-        SyndFeed feed = null;
-        try {
-            final SyndFeedInput input = new SyndFeedInput();
-/** for loading RSS feeds. 
- * User: ktlili
- * Date: Jan 5, 2010
- * Time: 4:07:04 PM
- */
-public class RSSUtil {
-    private static final transient Logger logger = org.slf4j.LoggerFactory.getLogger(RSSUtil.class);
-    
-    private SyndFeed feed;
-    
-    private boolean loaded;
-    private String url;
-    private static int timeoutInMs = 4000; // default value -> to be loaded from Jahia properties? (global timeout config)
-    
-    /**
-     * Get a SyndFeed from an url
-     * @param url
-     * @return
-     */
-    public static SyndFeed loadSyndFeed(String url){
-        SyndFeed feed = null;
-        try {
-            final SyndFeedInput input = new SyndFeedInput();
- /** for loading RSS feeds. 
- * User: ktlili
- * Date: Jan 5, 2010
- * Time: 4:07:04 PM
- */
-public class RSSUtil {
     private static final transient Logger logger = org.slf4j.LoggerFactory.getLogger(RSSUtil.class);
     
     private SyndFeed feed;
@@ -214,7 +143,8 @@ public class RSSUtil {
     /** 
     * Returns feed maximum timeout (to load).
     * @return the timeout value in milliseconds
-    */public static int getTimeoutInMs() {
+    */
+    public static int getTimeoutInMs() {
         return timeoutInMs;
     }
     /** 

